modules/tribes/client/components/TribesHeader.component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import Board from '../../../core/client/components/Board';
+
+export default function TribesHeader({ isLogged, onBoardChanged }) {
+
+  const { t } = useTranslation('tribes');
+
+  return <Board name="tribes-1" onNameChanged={onBoardChanged}>
+    <section className="board tribes-header" tr-boards="'tribes-1'">
```

Comment:
Can remove `tr-boards="'tribes-1'"` now it's in react :)

Reply:
Moved the whole `<section>` inside `Board` in fc4f0f4bdf7fa8a47447c9f638ac7c9c74ff6623